### ISSUE TITLE
 Hello! Author, I am very sorry for taking up your precious time. Recently, I am studying one of your papers-PraNet: Parallel Reverse Attention Network for Polyp Segmentation. In the process of learning PraNet network, there is a question that has puzzled me. The problem is as follows: In the Loss Function section of Section 2.3, ' the weighted IoU loss and binary cross entropy (BCE) loss for the global restriction and local (pixel-level) restriction.' What do "global restriction and local restriction" mean exactly? Thank you!

### ISSUE THREAD

**Issue by Progressiveyouth** [DengPingFan/PraNet#59]

**Comment by GewelsJI** [comment]
Good question here. You can understand them as:

- 'global' in weighted IOU loss means that IoU can focus on structural info
- 'local' in BCE loss means that pixel-wise restrictions between gt and pred

Best
